etl/command.py

```diff
@@ -301,21 +300,34 @@ def main(
     if grapher:
         sanity_check_db_settings()
 
-    dag = construct_dag(dag_path, private=private, grapher=grapher, export=export)
+    # Load DAG
+    dag = load_dag(dag_path)
+
+    # Create full DAG
```

Comment:
If I understand the intention here, load_dag should simply load the dag, regardless of our internal ETL logic, and construct_full_dag should apply additional logic. Is that right?
If so, I see that load_dag currently checks for "fasttrack" steps. Shouldn't this be part of construct_full_dag?